hack/tests/subcommand-scorecard.sh

```diff
@@ -20,7 +20,7 @@ operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep '^.*er
 
 # test to see if v1alpha2 is used from the command line
 commandoutput="$(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH_V1ALPHA2" 2>&1)"
-failCount=`echo $commandoutput | grep -o ": fail" | wc -l`
+failCount=`echo $commandoutput | grep -o "fail" | wc -l`
 expectedFailCount=3
```

Comment:
Hi @jmccormick2001, 

I could not find any test here with --selector -l (passing the selector) and checking that just the tests with the label X was executed as I could not find the test with the --list / -l as well. Should we not add it here?